README.md

```diff
@@ -15,20 +25,28 @@ The `main()` function starts two threads: `blinky_thread` and `idle_thread`. It
 
 ## Building and running
 
-1. Connect a USB cable between the USB port on the target and the host computer.
-1. Run this command to build the example project and program the microcontroller flash memory:
+1. Connect a USB cable between the USB port on the board and the host computer.
+1. Run the following command to build the example project and program the microcontroller flash memory:
+
+    * Mbed CLI 2
 
     ```bash
-    $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm
+    $ mbed-tools compile -m <TARGET> -t <TOOLCHAIN> --flash
```

Comment:
`mbed-tools` should support `--sterm`. Do we have a reason for removing it from the instructions?

Reply:
It should stay, other updates similar to this contain slightly more detail info, we should align these: see https://github.com/ARMmbed/mbed-os-example-error-handling/pull/73/files